core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -1881,6 +1881,31 @@ private void checkHavingAliasSameAsColumn(boolean upperAlias) {
     relFn(relFn).ok(expectedSql);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-4876">[CALCITE-4876]
+   * Converting RelNode to SQL with CalciteSqlDialect gets wrong result
+   * while EnumerableIntersect is followed by EnumerableLimit.</a>.
```

Comment:
Please remove the redundant  '.'  after the EnumerableLimit.

Reply:
OK, I've removed it.